reference/UiTPAS.v2.json

```diff
@@ -849,7 +862,95 @@
           "required": true,
           "description": "The id of the passholder"
         }
-      ]
+      ],
+      "get": {
+        "summary": "Get passholder school",
+        "operationId": "get-passholders-school",
+        "responses": {
+          "200": {
+            "description": "OK",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Organizer"
+                },
+                "examples": {
+                  "Example": {
+                    "value": {
+                      "id": "9c47936c-bdee-11eb-8529-0242ac130003"
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "401": {
+            "$ref": "#/components/responses/Unauthorized"
+          },
+          "403": {
+            "$ref": "#/components/responses/Forbidden"
+          },
+          "404": {
+            "description": "Not found. Possbile error types:\n\n* https://api.publiq.be/probs/uitpas/passholder-not-found\n* https://api.publiq.be/probs/uitpas/school-not-found\n\nThe detail property might include more information for the client developer.\n",
```

Comment:
`https://api.publiq.be/probs/uitpas/school-not-found`

Wanneer kan dit voorvallen? Aangezien je enkel de passholder id meegeeft in de URL.

Reply:
Ja, hier heb ik nog over zitten twijfelen, maar dat is als je de pashouder school opvraagt, maar er geen ingesteld is...

Reply:
Ah right makes sense! 👍